### ISSUE TITLE
Main menu : Make it look more definite

### ISSUE THREAD

**Issue by Bertram25** [OpenDungeons/OpenDungeons#6]
The following options should be added:
- Campaign (only here the campaign list is needed, this gamemode would be the only one to call upon it; possibility to hold more than one campaign, just like Battle for Wesnoth does)
- Skirmish (all levels from the game are accessible from a list here, new ones can be added by finding and adding to the list)
- Multiplayer
- Load (a saved game) Technically, a saved game is a level file stored in the user save/ folder.
- Editor (Opens a menu permitting to create a new level, then a windows appears asking for size, biome, allowed creatures…; or pick one of a list to modify or search for one in your pc.)
- Configuration
- Credits
- Quit

Buttons not functional should be set to disabled.

**Comment by Bertram25** [comment]
@hwoarangmy 
@akien-mageia 
While seeing this issue, I was thinking we're very close from closing it. 

Atm we have Single-Player, Multiplayer, Editor, and quit. (The credits button is less important atm and can be described in another issue.)
IMHO, the Load button described above should be moved into the corresponding game types, and removed from the future main menu.

Thus my first question is, do you agree with the following main menu?
- Campaign (disabled atm)
- Skirmish (aka Single-Player)
- Multiplayer
- Editor
- Credits (disabled atm)
- Quit.

The Options button could be moved in a corner as it is done in game, so the player can visually know it is the same button.

Would you agree then to move singleplayer maps to skirmish/ ?
We would then have a rather definite main menu and could move on from that.

What do you think?

**Comment by akien-mga** [comment]
I like your proposal, it's straight-forward and functional. Campaign \o/

What is the rationale for having the Options menu accessible in-game but not in the main menu? Well the question actually boils down to "What can be configured through the options?". It's true that the graphics part is configured through the OGRE GUI before starting the main menu, so if the remaining options are key bindings, sound/music, etc., it seems ok to have them in-game but not necessarily in the main menu.

There might be some more general settings though that are worth having directly in the main menu: player name for Multiplayer mode, and probably others but none come to mind for now :-p

**Comment by Bertram25** [comment]
> What is the rationale for having the Options menu accessible in-game but not in the main menu?

I didn't mean to remove it from the main menu but to put it in a corner, just as it is done in game. So visually it's same kind of thing for the player. I say that because later we might want to turn this into an icon, for instance.

**Comment by akien-mga** [comment]
Ok, that sounds good :-)

**Comment by Bertram25** [comment]
For the current release, after thinking about this, I'd rather not show useless buttons.

So I'd like to let the menus as is while renaming Single-Player to Skirmish (in reference for the future).
I would also hide the options button seen in game as it is not functional.
And I would move singleplayer maps into the skirmish/ folder.

then, this issue can be closed and a new one can be opened when we add support for options, credits and campaigns.

Ok with it, @hwoarangmy @akien-mga ?

**Comment by akien-mga** [comment]
+1

**Comment by hwoarangmy** [comment]
Yes, I also believe it is better to not show not functionnal buttons as it will look too much like a bug

**Comment by Bertram25** [comment]
Ok, I'll handle that one if nobody else takes it.
